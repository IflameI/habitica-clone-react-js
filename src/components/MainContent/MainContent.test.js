import { shallow } from 'enzyme';
import { MainContent } from '../';

describe('MainContent', () => {
  it('should render correctly without props', () => {
    const component = shallow(<MainContent />);

    expect(component).toMatchSnapshot();
  });
});
