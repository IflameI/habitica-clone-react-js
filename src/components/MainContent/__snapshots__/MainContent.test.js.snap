// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent should render correctly without props 1`] = `
<section
  className="main-content"
>
  <div
    className="container"
  >
    <div
      className="main-content__top"
    />
    <div
      className="main-content__row"
    >
      <MainContentHabit />
      <MainContentDaily />
      <MainContentGoal />
      <MainContentReward />
      <ToastContainer
        autoClose={5000}
        closeButton={[Function]}
        closeOnClick={true}
        draggable={true}
        draggableDirection="x"
        draggablePercent={80}
        hideProgressBar={false}
        newestOnTop={false}
        pauseOnFocusLoss={true}
        pauseOnHover={true}
        position="top-right"
        role="alert"
        rtl={false}
        transition={[Function]}
      />
    </div>
  </div>
</section>
`;

exports[`Member should render correctly without props 1`] = `
<section
  className="main-content"
>
  <div
    className="container"
  >
    <div
      className="main-content__top"
    />
    <div
      className="main-content__row"
    >
      <MainContentHabit />
      <MainContentDaily />
      <MainContentGoal />
      <MainContentReward />
      <ToastContainer
        autoClose={5000}
        closeButton={[Function]}
        closeOnClick={true}
        draggable={true}
        draggableDirection="x"
        draggablePercent={80}
        hideProgressBar={false}
        newestOnTop={false}
        pauseOnFocusLoss={true}
        pauseOnHover={true}
        position="top-right"
        role="alert"
        rtl={false}
        transition={[Function]}
      />
    </div>
  </div>
</section>
`;
